index.html

```diff
@@ -2356,9 +2401,39 @@
       iframe.addEventListener('destroy', () => {
         screenQuad.material.uniforms.numTextures.value = 1;
       });
-      container.appendChild(iframe);
-
-      iframes.push(iframe);
+        // function to test the URL on submit for a 404
+        function checkURL() {
+            let request;
+            if(window.XMLHttpRequest)
+                request = new XMLHttpRequest();
+            else
+                request = new ActiveXObject("Microsoft.XMLHTTP");
```

Comment:
I don't think we're supporting any browser with no `XMLHttpRequest` support. If it doesn't support that it probably can't draw anything anyway. :P